tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -309,6 +312,60 @@ def test_anomaly_detection_alert_seer_timeout_max_retry(self, mock_logger, mock_
         assert resp.data["detail"] == "Timeout error when hitting Seer store data endpoint"
         assert mock_seer_request.call_count == 1
 
+    @with_feature("organizations:anomaly-detection-alerts")
+    @with_feature("organizations:incidents")
+    @patch(
+        "sentry.seer.anomaly_detection.store_data.seer_anomaly_detection_connection_pool.urlopen"
+    )
+    def test_anomaly_detection_alert_not_enough_data(self, mock_seer_request):
```

Comment:
This test mysteriously started failing and just as mysteriously began working again... nothing about it looks flaky, as far as I can tell, but maybe there's something to do with time here that I'm missing?